src/lib/components/Router.svelte

```diff
@@ -2,24 +2,21 @@
   import type Router from "$lib/classes/MPLS/Router";
 
   export let router: Router;
-
-  const colorMap = {
-    CE: "#7FC8F8",
-    LER: "#FFE45E",
-    LSR: "#FF6392",
-  };
 </script>
 
-<circle
-  id={router.id.toString()}
-  cx={router.node.x}
-  cy={router.node.y}
-  r="20"
-  fill={colorMap[router.type]}
-/>
-<text x={router.node.x} y={router.node.y} dominant-baseline="central" font-size="smaller">
-  {router.node.label}
-</text>
+
+
+  <image
+    id={router.id.toString()}
+    x={router.node.x - 20}
+    y={router.node.y - 20}
+    width="45"
+    height="45"
+    href={`/${router.type}Router.svg`}
```

Comment:
this should be a RouterSVG.svelte file
aka look at this for reference: https://svelte.dev/repl/86553af5c35449ab88e34f1e50eb298f?version=3.38.0

Reply:
and don't use image tag